app/views/schools/add_participants/choose_mentor.html.erb

```diff
@@ -5,9 +5,10 @@
 <span class="govuk-caption-l"><%= @school.name %>, <%= @cohort.display_name %> cohort</span>
 
 <%= form_for add_participant_form, url: { action: :update }, method: :patch do |f| %>
-<%= f.govuk_radio_buttons_fieldset :mentor_id,
-  legend: { text: "Who is the mentor for this Early Career Teacher?", tag: 'h1', size: 'xl' },
-  hint: { text: "If you don't know, or haven't added them yet you can do this later." } do %>
+  <%= f.hidden_field :mentor_id, value: '' %>
```

Comment:
Why do we need this hidden field?

Reply:
Without a hidden field, no params are sent on the form submission when no option is selected. This caused some issues with strong params (`school_add_participant_form param missing or empty` exception). Even if we skip strong params in those cases, there was a bit of an edge case when user comes back to this page (after submitting it previously), un-selects the radio and submits the form: since it subitted empty params the form is not changed, remains valid and proceeds forward without any change. It is really an edge case as it is not that easy to unselect a radio, but I thought it is probably best to have this fixed as well. If we were to use `govuk_radiu_buttons_collection` it would add that field automatically.